test/test_config_params.py

```diff
@@ -25,6 +25,12 @@ class Inherited(gokart.TaskOnKart):
     param_b = luigi.Parameter(default='overrided')
 
 
+@inherits_config_params(ConfigClass, param_config2task={'param_a': 'param_d'})
```

Comment:
When would you be happy with this feature?

Reply:
@vaaaaanquish 
This would be useful when `ConfigClass` and Wrapped class has different parameter name but want to have common value.

Reply:
@vaaaaanquish 
I've added use case to the docs.
https://github.com/m3dev/gokart/pull/193/files#diff-bd31f137cd4066a176d3de6d9729841a660ed1416d3a26cf9bd0c820211ad8b8R57